spec/requests/api/v0/articles_spec.rb

```diff
@@ -1174,4 +1174,89 @@ def put_article(**params)
       end
     end
   end
+
+  describe "GET /api/articles/search" do
+    before { article }
+
+    it "returns CORS headers" do
+      origin = "http://example.com"
+      get "/api/articles/search", headers: { origin: origin }
```

Comment:
I would much prefer to use a path where we have "/api/articles/search", but something funky was happening with the path that I didn't figure out. Since we're hoping to merge this and publish the plugin, I've added a cleanup taks for myself to fix this in a subsequent PR.